tests/integration/plugins/languages/is-supported.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+
+/** @import {SupportLanguage} from "prettier" */
+
+/** @type {SupportLanguage[]} */
+export const languages = [
+  {
+    name: "language-name-does-not-matter",
+    parsers: ["parser-name-inferred-from-language-is-supported"],
+    isSupported: (file) =>
+      /^\.husky[\\/][^\\/]+/u.test(file) ||
+      /([\\/])\.husky\1[^\\/]+$/u.test(file),
```

Comment:
Not important, but I thought we had a rule for this pattern.

```suggestion
      /(?<separator>[\\/])\.husky\k<separator>[^\\/]+$/u.test(file),
```

Reply:
Thanks! I didn't know about `\k<separator>` syntax before!